### PR TITLE
Add painkillers and antidepressant pills + canisters

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Adds pills of the medicines added on #1243.

## Why / Balance
I plan to use some of them for round start for the psychologist + convenience for future loadout changes in case of chronic pain or depressive characters.

## Technical details
Added a YAML.

## Media
Nope.

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**
No CL.

### CODE REVIEW

**Comment by deltanedas** [comment]
mapped gamer stun pills throw em in a grinder then inject

**Comment by Radezolid** [comment]
They aren't mapped... yet. But i do intend on giving a pill canister to the psych for traitor usage.

**Comment by Avalon-Proto** [comment]
> They aren't mapped... yet. But i do intend on giving a pill canister to the psych for traitor usage.

What's this I hear. Handing the pysch a traitor item of mislabeled pills hm?

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml [hunk 1]
```diff
@@ -0,0 +1,278 @@
+- type: entity
+  suffix: Serenitol 10u
+  parent: Pill
+  id: PillSerenitol
+  components:
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml]: why is every prototype in this file in a different fucking order

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+  - type: StorageFill
+    contents:
+    - id: PillSerenitol
+      amount: 5
+
+- type: entity
+  suffix: Neurozenium 10u
+  parent: Pill
+  id: PillNeurozenium
+  components:
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml]: real

**Review by MilonPL — CHANGES_REQUESTED** [review]
https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml (hunk 1 above)
Comment (Radezolid) [reply Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml]: lmao, copy pasted upstream pill and pill canister without looking and just found out they aren't following their own docs. Will fix.

**Comment by Radezolid** [comment]
wait oh god i fumbled the merge from master.

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml [hunk 3]
```diff
@@ -28,3 +28,251 @@
...
+  parent: PillCanister
+  id: PillCanisterAgonolexyne
+  suffix: Agonolexyne 5u, 5
+  components:
+  - type: Label
+    currentLabel: agonolexyne 5u
+  - type: StorageFill
+    contents:
+    - id: PillAgonolexyne
+      amount: 5
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml]: EOF newline

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Review by MilonPL — APPROVED** [review]

**Pull request merged**